### DISCUSSION TITLE
Change how transparent windows are implemented.

### DISCUSSION THREAD

**Discussion by onesounds** [Flow-Launcher/Flow.Launcher#737] [Ideas]
ModernWPF doesn't support Acrylic Effect window.

**References**
- https://github.com/sourcechord/FluentWPF
- https://github.com/AvaloniaUI/Avalonia

**Issues**
Windows 11 has problems. FL has same problem too.
- https://github.com/sourcechord/FluentWPF/issues/130
- https://github.com/Wox-launcher/Wox/issues/3541

When the above problem is solved, I will try to apply it again.

**What's Better**
- Speed up when using transparent windows.
- Natural shadow and transparent effect.
- The same window design as os.
- You can use shadow even when it's transparent theme

**Comment by jjw24** [comment]
is there an issue made at ModernWPF to find out if support will be added?

**Reply by onesounds** [reply]
Modernwpf is also required to use fluentwpf. Currently, fluentwpf also has a problem with translucent processing on Windows 11. It can be tested only when it is modified. I've considered using avalonia or winui3, but this is not what I can do. In addition, the priority was lowered because Windows 7 support continued to be mentioned.

When I was using Win 11, I knew there was a problem with the translucent function, so I thought it was a problem. There is no problem with Win10 users. So I gave up doing this work and lost interest in this related work. For now, I think the best option is to eliminate translucent themes.
